src/test/java/com/gdschongik/gdsc/domain/study/domain/StudyTest.java

```diff
@@ -0,0 +1,92 @@
+package com.gdschongik.gdsc.domain.study.domain;
+
+import static com.gdschongik.gdsc.domain.member.domain.Department.D022;
+import static com.gdschongik.gdsc.domain.member.domain.Member.createGuestMember;
+import static com.gdschongik.gdsc.global.common.constant.MemberConstant.DISCORD_USERNAME;
+import static com.gdschongik.gdsc.global.common.constant.MemberConstant.EMAIL;
+import static com.gdschongik.gdsc.global.common.constant.MemberConstant.NAME;
+import static com.gdschongik.gdsc.global.common.constant.MemberConstant.NICKNAME;
+import static com.gdschongik.gdsc.global.common.constant.MemberConstant.OAUTH_ID;
+import static com.gdschongik.gdsc.global.common.constant.MemberConstant.PHONE_NUMBER;
+import static com.gdschongik.gdsc.global.common.constant.MemberConstant.STUDENT_ID;
+import static com.gdschongik.gdsc.global.common.constant.MemberConstant.UNIV_EMAIL;
```

Comment:
```suggestion
import static com.gdschongik.gdsc.global.common.constant.MemberConstant.*;
```

아래에 있는 다른 상수들도 같이 수정부탁드립니다!